quickshop-bukkit/src/main/java/com/ghostchu/quickshop/menu/ShopStaffMenu.java

```diff
@@ -37,15 +39,19 @@ public class ShopStaffMenu extends QuickShopMenu {
 
   public static final String STAFF_PAGE = "STAFF_SELECTION_PAGE";
   public static final String PLAYER_PAGE = "PAGE_SELECTION_PAGE";
+  public static final String STAFF_SEARCH = "STAFF_SEARCH";
+  public static final String PLAYER_SEARCH = "PLAYER_SEARCH";
 
 
   public ShopStaffMenu() {
 
-    this.rows = 6;
+    // Load rows from config or use default (6 rows for modern layout)
+    final GuiConfig.MenuConfig menuConfig = QuickShop.getInstance().getGuiConfig().getMenuConfig("staff");
+    this.rows = menuConfig != null ? menuConfig.getRows() : 6;
```

Comment:
The ternary condition should have parentheses around it, with the question mark operator having no space before it.